src/storage/hummock_trace/src/collector.rs

```diff
@@ -316,6 +316,10 @@ impl TraceSpan {
         Self::new_global_op(Operation::Flush(delete_range), storage_type)
     }
 
+    pub fn new_try_flush_span(storage_type: StorageType) -> MayTraceSpan {
+        Self::new_global_op(Operation::Flush(vec![]), storage_type)
```

Comment:
nits: shouldn't this be a new operation type? otherwise, we cannot distinguish try flush with real flush.